spec/controllers/account_reset/delete_account_controller_spec.rb

```diff
@@ -29,6 +29,26 @@
       expect(response).to redirect_to account_reset_confirm_delete_account_url
     end
 
+    it 'logs a good token to the attempts api' do
+      user = create(:user, :signed_up, :with_backup_code)
+      create(:phone_configuration, user: user, phone: '+1 703-555-1214')
```

Comment:
If we're not using the specific phone number:
```suggestion
      create(:phone_configuration, user: user, phone: Faker::PhoneNumber.cell_phone)
```